package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3660,14 +3803,26 @@ def __init__(self, name, residues):
         self._cls = ResidueGroup
 
     @property
+    @deprecate(message="{}; use `segid` property instead".format(_FIFTEEN_DEPRECATION))
```

Comment:
This is important to me. When we issue deprecation messages then the suggested alternative should work.